react/react-training/src/app/components/AppRoutes/index.js

```diff
@@ -1,19 +1,42 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import connect from 'react-redux/es/connect/connect';
 import { Switch, Route } from 'react-router';
 import { BrowserRouter as Router } from 'react-router-dom';
 import Game from '@screens/Game';
 import Login from '@screens/Login';
 import { ROUTES } from '@constants/routes.js';
 
-export default class AppRoutes extends Component {
+import AuthRoute from './components/AuthRoute';
+
+class AppRoutes extends Component {
   render() {
+    let { userIsLogged } = this.props;
+    if (!userIsLogged) {
+      userIsLogged = localStorage.getItem('userIsLogged') === 'true';
+    }
     return (
       <Router>
         <Switch>
-          <Route exact path={ROUTES.HOME} component={Login} />
+          <AuthRoute exact path={ROUTES.HOME} component={Login} userIsLogged={userIsLogged} />
           <Route exact path={ROUTES.GAME} component={Game} />
```

Comment:
What happen if you're in `Login` page and in the URL you try access to `Game` component?